README.md

```diff
@@ -97,6 +97,22 @@ CONVERSATION_PASSWORD=<add_conversation_password>
 WORKSPACE_ID=<add_conversation_workspace>
 ```
 
+### Authentication using IAM
+
+The Watson Assistant service has a new API authentication process. You use IAM for authentication. You can pass either a bearer token or an API key. Tokens support authenticated requests without embedding service credentials in every call. API keys use basic authentication.
```

Comment:
I think this is confusing, since not all regions have this "new" auth process....
Maybe we should just leave this out, and show both images, and say:
if you have uname/password, use them
if you have api_key, use it.